lib/cc/cli/analyze.rb

```diff
@@ -40,6 +40,8 @@ def process_args
             @engine_options << @args.shift
           when "--dev"
             @dev_mode = true
+          when "--timeout"
+            ENV["CONTAINER_TIMEOUT_SECONDS"] = @args.shift
```

Comment:
@GordonDiggs good call.

Thought I saw a case where ENV wasn't working but I think it was just that the engine ran for less than 1 second. 

Thanks!